tests/phpunit/tests/comment.php

```diff
@@ -927,6 +927,8 @@ public function test_wp_get_comment_fields_max_lengths() {
 
 		$lengths = wp_get_comment_fields_max_lengths();
 
+		$this->assertNotEmpty( $lengths );
```

Comment:
As `$lengths` is subsequently used in a foreach, might it be an idea to also add an `assertIsArray( $lengths )` or `assertIsIterable( $lengths )` as well ? (here and in quite a few other places)

Something not being "empty", does not guarantee that it's an iterable array after all.

Reply:
Possibly, although I think they protect against different things. My proposed change protects against the `foreach` iterating an empty array resulting in the test performing no assertions.

If `$lengths` is not iterable, we'll get a PHP warning which will cause the test to fail, so I don't think `assertIsArray` is necessary here.